builder/commands_test.go

```diff
@@ -0,0 +1,27 @@
+package builder
+
+import (
+	"testing"
+	"time"
+
+	"github.com/Masterminds/cookoo"
+)
+
+func TestSleep(t *testing.T) {
+	reg, router, cxt := cookoo.Cookoo()
+
+	reg.Route("test", "Test route").
+		Does(Sleep, "res").Using("duration").WithDefault(3 * time.Second)
+
+	start := time.Now()
+	err := router.HandleRequest("test", cxt, true)
+	if err != nil {
+		t.Error(err)
+	}
+	end := time.Now()
+
+	if end.Sub(start) < 3*time.Second {
+		t.Error("Expected elapsed time to be 3 seconds.")
```

Comment:
Error message should not be capitalized. https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Also, this does not tell us what was the actual time, why don't you use the want, got pattern?

```
if want, got := end.Sub(start), 3*time.Second; want < got {
    t.Errorf("expected elapsed time to be at least %s got %s", want, got)
}
```